filter/filter.go

```diff
@@ -478,6 +479,33 @@ func (c *Client) GetDimensionOptionsBatchProcess(ctx context.Context, userAuthTo
 	return eTag, batch.ProcessInConcurrentBatches(batchGetter, batchProcessor, batchSize, maxWorkers)
 }
 
+// DeleteDimensionOptions completely removes the options array from a given dimension
+func (c *Client) DeleteDimensionOptions(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, filterID, name string) (eTag string, err error) {
+
+	uri := fmt.Sprintf("%s/filters/%s/dimensions/%s/options", c.hcCli.URL, filterID, name)
+	clientlog.Do(ctx, "deleting selected dimension options", service, uri)
+
+	resp, err := c.doDeleteWithAuthHeadersAndWithDownloadToken(ctx, userAuthToken, serviceAuthToken, collectionID, uri)
+
+	if err != nil {
```

Comment:
Please wrap errors (and remove this blank line between checking the error)

Reply:
This has been addressed.